package.json

```diff
@@ -11,7 +11,7 @@
     "test": "mocha",
     "cover": "rm -Rf coverage/* && istanbul cover _mocha -- -R spec",
     "lint": "eslint bin/* lib/*.js lib/**/*.js test/*.js",
-    "install": "cd pkcs7 && bundle install --path='./'",
+    "install": "cd pkcs7 && bundle package",
```

Comment:
`bundle package` != `bundle install` The pacakge command just downloads the required `gem` artifacts to `vendor/cache`. It doesn't install them in a way that the ruby process can then load.